src/login/LoginForm.test.js

```diff
@@ -26,33 +27,37 @@ describe('login page', () => {
     ['', 'test', true],
     ['', '', true],
   ])('check button availability for user: %s, pass: %s', (username, password, expected) => {
-    render(<Login />);
-    const usernameInput = screen.getByLabelText(/username/);
-    const passwordInput = screen.getByLabelText(/password/);
+    render(<LoginForm />);
+    const usernameInput = screen.getByLabelText(/Username/);
+    const passwordInput = screen.getByLabelText(/Password/);
     const loginBtn = screen.getByRole('button');
     userEvent.type(usernameInput, username);
     userEvent.type(passwordInput, password);
     expect(loginBtn.hasAttribute('disabled')).toEqual(expected);
   });
 
   it('check login button', async () => {
-    render(<Login />);
+    render(<LoginForm />);
 
-    const usernameInput = screen.getByLabelText(/username/);
-    const passwordInput = screen.getByLabelText(/password/);
+    const usernameInput = screen.getByLabelText(/Username/);
+    const passwordInput = screen.getByLabelText(/Password/);
     const loginBtn = screen.getByRole('button');
+
     const wrongInput = 'test';
-    const spy = jest.spyOn(window, 'alert').mockImplementation();
     userEvent.type(usernameInput, wrongInput);
     userEvent.type(passwordInput, wrongInput);
     userEvent.click(loginBtn);
-    expect(spy).toHaveBeenCalledWith('Login Failed');
+    await new Promise((resolve) => setTimeout(resolve, 0));
+    expect(await screen.findByTestId('login--info-alert')).toBeInTheDocument();
+
+    const correctUsernameInput = 'User';
+    const correctPasswordInput = 'UXdlcnR5MTIz';
     userEvent.clear(usernameInput);
     userEvent.clear(passwordInput);
-    const correctInput = 'admin';
-    userEvent.type(usernameInput, correctInput);
-    userEvent.type(passwordInput, correctInput);
+    userEvent.type(usernameInput, correctUsernameInput);
+    userEvent.type(passwordInput, correctPasswordInput);
     userEvent.click(loginBtn);
-    expect(spy).toHaveBeenCalledWith('Hooray! You just logged in');
+    await new Promise((resolve) => setTimeout(resolve, 0));
```

Comment:
what does this achieve?

Reply:
Honestly this was some recommendation from Apollo Docs 😄